android/src/main/java/com/attentivereactnativesdk/AttentiveReactNativeSdkModule.java

```diff
@@ -244,4 +307,257 @@ private List<Item> buildItems(ReadableArray rawItems) {
+    contentContainer.setBackgroundColor(0xFFFFFFFF);
+    contentContainer.setPadding(24, 24, 24, 24);
+
+    // Round corners
+    android.graphics.drawable.GradientDrawable background = new android.graphics.drawable.GradientDrawable();
+    background.setColor(0xFFFFFFFF);
+    background.setCornerRadius(24);
+    contentContainer.setBackground(background);
+
+    // Header with title and close button
+    android.widget.LinearLayout headerLayout = new android.widget.LinearLayout(activity);
+    headerLayout.setOrientation(android.widget.LinearLayout.HORIZONTAL);
+    headerLayout.setGravity(android.view.Gravity.CENTER_VERTICAL);
+
+    android.widget.TextView titleText = new android.widget.TextView(activity);
+    titleText.setText("🐛 Attentive Debug Session");
+    titleText.setTextSize(18);
+    titleText.setTypeface(null, android.graphics.Typeface.BOLD);
+    titleText.setTextColor(0xFF000000); // Black text
+    android.widget.LinearLayout.LayoutParams titleParams = new android.widget.LinearLayout.LayoutParams(0, android.widget.LinearLayout.LayoutParams.WRAP_CONTENT, 1);
+    titleText.setLayoutParams(titleParams);
+    headerLayout.addView(titleText);
+
+    // X Close button
+    android.widget.Button closeButton = new android.widget.Button(activity);
+    closeButton.setText("✕");
+    closeButton.setTextSize(18);
+    closeButton.setTextColor(0xFF666666); // Dark gray text
+    closeButton.setBackgroundColor(0xFFF0F0F0);
+    closeButton.setPadding(16, 8, 16, 8);
+    android.widget.LinearLayout.LayoutParams closeParams = new android.widget.LinearLayout.LayoutParams(
+      android.widget.LinearLayout.LayoutParams.WRAP_CONTENT,
+      android.widget.LinearLayout.LayoutParams.WRAP_CONTENT);
+    closeButton.setLayoutParams(closeParams);
+    headerLayout.addView(closeButton);
+
+    contentContainer.addView(headerLayout);
+
+    // Tab buttons
+    android.widget.LinearLayout tabLayout = new android.widget.LinearLayout(activity);
+    tabLayout.setOrientation(android.widget.LinearLayout.HORIZONTAL);
+    tabLayout.setPadding(0, 20, 0, 0);
+
+    android.widget.Button currentTab = new android.widget.Button(activity);
+    currentTab.setText("Current Event");
+    currentTab.setTextColor(0xFF000000); // Black text
+    currentTab.setLayoutParams(new android.widget.LinearLayout.LayoutParams(0, android.widget.LinearLayout.LayoutParams.WRAP_CONTENT, 1));
+
+    android.widget.Button historyTab = new android.widget.Button(activity);
+    historyTab.setText("History (" + debugHistory.size() + ")");
+    historyTab.setTextColor(0xFF000000); // Black text
+    historyTab.setLayoutParams(new android.widget.LinearLayout.LayoutParams(0, android.widget.LinearLayout.LayoutParams.WRAP_CONTENT, 1));
+
+    tabLayout.addView(currentTab);
+    tabLayout.addView(historyTab);
+    contentContainer.addView(tabLayout);
+
+    // Content area
+    android.widget.ScrollView contentScroll = new android.widget.ScrollView(activity);
```

Comment:
Can we get rid of all the explicit imports and just include at the top of the file?